flink-examples/flink-examples-table/pom.xml

```diff
@@ -53,34 +53,42 @@ under the License.
 			<version>${project.version}</version>
 		</dependency>
 
-		<!-- Table connectors and formats -->
+		<!-- The following two dependencies are not required to define a SQL job pipeline,
+		but only to execute it.
+
+		In particular, here we're forced to use flink-table-planner_${scala.binary.version} instead of
+		flink-table-planner-loader, because otherwise you hit this bug https://youtrack.jetbrains.com/issue/IDEA-93855
+		when trying to run the examples from within Intellij IDEA.
+
+		In a real environment, you need these dependencies only at test scope, to execute the SQL job pipelines test.
+		 -->
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-connector-files</artifactId>
+			<artifactId>flink-table-runtime</artifactId>
 			<version>${project.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-csv</artifactId>
+			<artifactId>flink-table-planner_${scala.binary.version}</artifactId>
```

Comment:
Provided? Or is it somehow excluded from the JAR files we generate for the dist.

Reply:
I think it makes no difference, as this pom doesn't produce a fat jar. In any case, I put it as provided more for documentation purpose.